<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Pere Constans</author>
 <class>c2bReferenceList</class>
 <widget class="QWidget" name="c2bReferenceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>References</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="2" column="2">
    <widget class="QToolButton" name="clearB">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>Clear filter</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="cb2bib.qrc">
       <normaloff>:/icons/icons/clear_left.png</normaloff>:/icons/icons/clear_left.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="filterB">
     <property name="statusTip">
      <string>Filter Author and Title</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="cb2bib.qrc">
       <normaloff>:/icons/icons/filter.png</normaloff>:/icons/icons/filter.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTreeWidget" name="listView">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>0</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>5</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="c2bComboBox" name="comboFilter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>c2bComboBox</class>
   <extends>QComboBox</extends>
   <header>c2bComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cb2bib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearB</sender>
   <signal>clicked()</signal>
   <receiver>comboFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
